routes/web.php

```diff
@@ -13,10 +17,27 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+// Route::get('/', function () {
+//     return view('welcome');
+// });
+
+
 
 Auth::routes();
 
-Route::get('/home', [App\Http\Controllers\HomeController::class, 'index'])->name('home');
+Route::get('/', [HomeController::class, 'create']);
+
+Route::get('/home', [HomeController::class, 'index'])->name('home');
+
+
+
+Route::get('/create-category', [CategoryController::class, 'create'])->name('category');
+Route::post('/create-category', [CategoryController::class, 'store'])->name('storeCategory');
+Route::get('/all-categories', [CategoryController::class, 'index'])->name('allCategory');
+Route::get('/destroy-category/{id}', [CategoryController::class, 'destroy'])->name('destroy');
+
+
+Route::get('/create-product', [ProductController::class, 'create'])->name('createProduct');
```

Comment:
Here too @Ochegbinu do something like `products/create`. insteate of create-product because you will group your routes later

Reply:
Done too sir